src/utils.js

```diff
@@ -34,7 +34,7 @@ export function isStoryPublic(story) {
 }
 
 export function getWatermarkImage(story, cdnSrc, cdnURL) {
-  const watermarkImageS3Key = get(story, ["watermark", "social", "image-s3-key"], "");
+  const watermarkImageS3Key = get(story, ["watermark", "social", "image-s3-key"], false);
   if (cdnSrc && cdnSrc.includes("gumlet") && watermarkImageS3Key.length > 0) {
```

Comment:
@shraddha is this working fine , since **string** has changed to **boolean**,
 We may have to refactor line 38 too , right
`if (cdnSrc && cdnSrc.includes("gumlet") && watermarkImageS3Key.length > 0) {`

Reply:
It is working fine.

Reply:
![Screenshot 2023-06-21 at 11 20 23 AM](https://github.com/quintype/quintype-node-seo/assets/104062026/07ef5af7-75cd-40e1-9e3b-1447503a79d3)